docs/src/modules/components/AppFrame.js

```diff
@@ -229,16 +226,14 @@ function AppFrame(props) {
               onClick={handleLanguageIconClick}
               data-ga-event-category="AppBar"
               data-ga-event-action="language"
+              startIcon={<LanguageIcon fontSize="small" />}
```

Comment:
I'm not sure about the change:

Before:

![Capture d’écran 2019-09-29 à 18 42 28](https://user-images.githubusercontent.com/3165635/65835622-f1e33480-e2e8-11e9-83a4-aa0a2ce1c64f.png)

After:

![Capture d’écran 2019-09-29 à 18 42 31](https://user-images.githubusercontent.com/3165635/65835623-f1e33480-e2e8-11e9-8237-2d654428de62.png)

Reply:
Oops, I should have tried it! I'll partially revert.